app/controllers/catalog_controller.rb

```diff
@@ -21,6 +21,10 @@ class CatalogController < ApplicationController
     redirect_to solr_document_path(params[:id])
   end
 
+  rescue_from BlacklightRangeLimit::InvalidRange do
+    redirect_to '/', :flash => { :error => 'The min year must be less than the max year.' }
```

Comment:
I think we shouldn't use "min year" and "max year".  There are no labels to guide the user, and that wording doesn't seem like what I would expect a user to use.  Can you change the error message to be something like "The start year must be before the end year"?  I think that wording will be more easily understandable.

Reply:
ok!